Microsoft.Toolkit.Uwp/Helpers/PrintHelper/PrintHelperOptions.cs

```diff
@@ -0,0 +1,187 @@
+        /// </summary>
+        public IList<string> DisplayedOptions { get; private set; }
+
+        /// <summary>
+        /// Gets the possible display options
+        /// </summary>
+        private IEnumerable<string> _possibleDisplayOptions;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the default displayed options should be kept.
+        /// Defaults to true
+        /// </summary>
+        public bool ExtendDisplayedOptions { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PrintHelperOptions"/> class.
+        /// </summary>
+        /// <param name="extendDisplayedOptions">Boolean used to set up the <see cref="ExtendDisplayedOptions"/> property</param>
+        public PrintHelperOptions(bool extendDisplayedOptions = true)
+        {
+            ExtendDisplayedOptions = extendDisplayedOptions;
+            DisplayedOptions = new List<string>();
+
+            InitializePossibleDisplayOptions();
+        }
+
+        /// <summary>
+        /// Adds a display option
+        /// </summary>
+        /// <param name="displayOption">Display option to add. Must be a part of the <see cref="StandardPrintTaskOptions"/> class</param>
+        public void AddDisplayOption(string displayOption)
+        {
+            if (!_possibleDisplayOptions.Contains(displayOption))
+            {
+                throw new ArgumentException("The display option must be a part of the StandardPrintTaskOptions class");
+            }
+
+            if (DisplayedOptions.Contains(displayOption))
+            {
+                return;
+            }
+
+            DisplayedOptions.Add(displayOption);
+        }
+
+        /// <summary>
+        /// Removes a display option
+        /// </summary>
+        /// <param name="displayOption">Display option to add. Must be a part of the <see cref="StandardPrintTaskOptions"/> class</param>
+        public void RemoveDisplayOption(string displayOption)
+        {
+            if (!_possibleDisplayOptions.Contains(displayOption))
+            {
+                throw new ArgumentException("The display option must be a part of the StandardPrintTaskOptions class");
+            }
+
+            if (!DisplayedOptions.Contains(displayOption))
+            {
+                return;
+            }
+
+            DisplayedOptions.Remove(displayOption);
+        }
+
+        private void InitializePossibleDisplayOptions()
+        {
+            _possibleDisplayOptions = typeof(StandardPrintTaskOptions).GetProperties(BindingFlags.Public | BindingFlags.Static | BindingFlags.FlattenHierarchy)
+                                                                      .Where(p => p.PropertyType == typeof(string))
+                                                                      .Select(p => (string)p.GetValue(null));
+        }
+
+        private void InitializeDefaultOptions()
+        {
+            NumberOfCopies = 0;
```

Comment:
Shouldn't be 1 instead of 0?

Reply:
It's a default value. That way I know if the property was modified and should replace the value loaded in PrintTaskOptions.

Reply:
Actually, I forgot to add the property to PrintHelper. After adding it, the app crashed when setting it. I decided to remove it completely.